// @pv-frontend/pv-components-library/fesm2020/pv-frontend-pv-components-library.mjs
import * as i0 from '@angular/core';
import { Injectable, Component, NgModule } from '@angular/core';

class PvComponentsLibraryService {
    constructor() { }
}
PvComponentsLibraryService.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "15.2.9", ngImport: i0, type: PvComponentsLibraryService, deps: [], target: i0.ɵɵFactoryTarget.Injectable });
PvComponentsLibraryService.ɵprov = i0.ɵɵngDeclareInjectable({ minVersion: "12.0.0", version: "15.2.9", ngImport: i0, type: PvComponentsLibraryService, providedIn: 'root' });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "15.2.9", ngImport: i0, type: PvComponentsLibraryService, decorators: [{
            type: Injectable,
            args: [{
                    providedIn: 'root'
                }]
        }], ctorParameters: function () { return []; } });

class PvComponentsLibraryComponent {
}
PvComponentsLibraryComponent.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "15.2.9", ngImport: i0, type: PvComponentsLibraryComponent, deps: [], target: i0.ɵɵFactoryTarget.Component });
PvComponentsLibraryComponent.ɵcmp = i0.ɵɵngDeclareComponent({ minVersion: "14.0.0", version: "15.2.9", type: PvComponentsLibraryComponent, selector: "lib-pv-components-library", ngImport: i0, template: `
    <p>
      pv-components-library works!
    </p>
  `, isInline: true });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "15.2.9", ngImport: i0, type: PvComponentsLibraryComponent, decorators: [{
            type: Component,
            args: [{ selector: 'lib-pv-components-library', template: `
    <p>
      pv-components-library works!
    </p>
  ` }]
        }] });

class ButtonComponent {
}
ButtonComponent.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "15.2.9", ngImport: i0, type: ButtonComponent, deps: [], target: i0.ɵɵFactoryTarget.Component });
ButtonComponent.ɵcmp = i0.ɵɵngDeclareComponent({ minVersion: "14.0.0", version: "15.2.9", type: ButtonComponent, selector: "lib-button", ngImport: i0, template: "<button>Click me!!!</button>\n", styles: [""] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "15.2.9", ngImport: i0, type: ButtonComponent, decorators: [{
            type: Component,
            args: [{ selector: 'lib-button', template: "<button>Click me!!!</button>\n" }]
        }] });

class CardComponent {
}
CardComponent.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "15.2.9", ngImport: i0, type: CardComponent, deps: [], target: i0.ɵɵFactoryTarget.Component });
CardComponent.ɵcmp = i0.ɵɵngDeclareComponent({ minVersion: "14.0.0", version: "15.2.9", type: CardComponent, selector: "lib-card", ngImport: i0, template: "<div>\n  <h3>Zomato</h3>\n  <p>50% Offer on Zomato Membership</p>\n</div>\n", styles: [""] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "15.2.9", ngImport: i0, type: CardComponent, decorators: [{
            type: Component,
            args: [{ selector: 'lib-card', template: "<div>\n  <h3>Zomato</h3>\n  <p>50% Offer on Zomato Membership</p>\n</div>\n" }]
        }] });

class PvComponentsLibraryModule {
}
PvComponentsLibraryModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "15.2.9", ngImport: i0, type: PvComponentsLibraryModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
PvComponentsLibraryModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "14.0.0", version: "15.2.9", ngImport: i0, type: PvComponentsLibraryModule, declarations: [PvComponentsLibraryComponent,
        ButtonComponent,
        CardComponent], exports: [PvComponentsLibraryComponent,
        ButtonComponent,
        CardComponent] });
PvComponentsLibraryModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "15.2.9", ngImport: i0, type: PvComponentsLibraryModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "15.2.9", ngImport: i0, type: PvComponentsLibraryModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [
                        PvComponentsLibraryComponent,
                        ButtonComponent,
                        CardComponent
                    ],
                    imports: [],
                    exports: [
                        PvComponentsLibraryComponent,
                        ButtonComponent,
                        CardComponent
                    ]
                }]
        }] });

/*
 * Public API Surface of pv-components-library
 */

/**
 * Generated bundle index. Do not edit.
 */

export { ButtonComponent, CardComponent, PvComponentsLibraryComponent, PvComponentsLibraryModule, PvComponentsLibraryService };
