// @pv-frontend/pv-components-library/esm2020/lib/pv-components-library.module.mjs
import { NgModule } from '@angular/core';
import { PvComponentsLibraryComponent } from './pv-components-library.component';
import { ButtonComponent } from './button/button.component';
import { CardComponent } from './card/card.component';
import * as i0 from "@angular/core";
export class PvComponentsLibraryModule {
}
PvComponentsLibraryModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "15.2.9", ngImport: i0, type: PvComponentsLibraryModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
PvComponentsLibraryModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "14.0.0", version: "15.2.9", ngImport: i0, type: PvComponentsLibraryModule, declarations: [PvComponentsLibraryComponent,
        ButtonComponent,
        CardComponent], exports: [PvComponentsLibraryComponent,
        ButtonComponent,
        CardComponent] });
PvComponentsLibraryModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "15.2.9", ngImport: i0, type: PvComponentsLibraryModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "15.2.9", ngImport: i0, type: PvComponentsLibraryModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [
                        PvComponentsLibraryComponent,
                        ButtonComponent,
                        CardComponent
                    ],
                    imports: [],
                    exports: [
                        PvComponentsLibraryComponent,
                        ButtonComponent,
                        CardComponent
                    ]
                }]
        }] });
